src/internal/operations/restore.go

```diff
@@ -159,10 +162,12 @@ func (op *RestoreOperation) Run(ctx context.Context) (restoreDetails *details.De
 		},
 	)
 
-	paths, err := formatDetailsForRestoration(ctx, op.Selectors, deets)
+	paths, err := formatDetailsForRestoration(ctx, op.Selectors, deets, op.Errors)
 	if err != nil {
-		opStats.readErr = err
-		return nil, err
+		op.Errors.Fail(err)
+		opStats.readErr = op.Errors.Err()
```

Comment:
Is there a change that `op.Errors` is nil?

Reply:
None